ivy/functional/backends/paddle/layers.py

```diff
@@ -435,4 +435,62 @@ def conv_general_transpose(
     bias: Optional[paddle.Tensor] = None,
     out: Optional[paddle.Tensor] = None,
 ):
-    raise IvyNotImplementedException()
+    if data_format == "channel_last":
```

Comment:
Thank you for your contribution! 🙌
Could you please add a unit test for the function?
Thanks!

Reply:
Thanks for the update. If you don't mind, can you shed some light on where to write the unit test for this backend function? I'm scouring the doc but am struggling a little bit.

Reply:
Sorry, my mistake. We already have an Ivy test!